### PR TITLE
W-14792360 configure a network proxy gr

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by nledesma — APPROVED** [review]

File: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
+= Configuring a Network Proxy for PDK
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+If you use a network proxy to connect to the internet, it may be necessary to define the network proxy for the PDK tools.
```
Comment (dhirsch1) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc]:
```suggestion
If you use a network proxy to connect to the internet, you might have to define the network proxy for the PDK tools.
```

Comment (dhirsch1) [reply pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc]: Is it obvious to the user when he/she must do this? If not, can you provide an example of when a user has to do this?

File: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Besides Docker, all tools downloaded in the xref:policies-pdk-prerequisites.adoc[PDK prerequisites] accept the `HTTP_PROXY` and `HTTPS_PROXY` environment variables. 
+
+To configure your environment variable, refer to <<configure-a-network-proxy>>.
```
Comment (dhirsch1) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc:11]: Combine this lead-in with the next one. 
To configure your environment variable or use Docker with a network proxy...

File: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+To use Docker with a network proxy, also <<configure-a-network-proxy-for-docker>>.
+
+
+[[configure-a-network-proxy]]
+== Configure a Network Proxy
+
+Configure the environment variables by running following relevant command after replacing the variables with your information:
```
Comment (dhirsch1) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc]:
```suggestion
Configure the environment variables by running following command after replacing the variables with your information:
```

File: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+----
+
+If your network proxy requires authentication, use the following command:
+
+[source,ssh]
+----
+export HTTPS_PROXY=<proxy-method>://<username>:<password>@<proxy-server>:<proxy-port>
+----
+
+The PDK tools may visit the following domains during execution:
```
Comment (dhirsch1) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc]: may > might

"visit the following domains" sounds funny: 
  "execute in the following domains"???
  "use the following domains"???

File: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc [hunk 5]
```diff
@@ -0,0 +1,79 @@
+= Configuring a Network Proxy for PDK
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+If you use a network proxy to connect to the internet, it may be necessary to define the network proxy for the PDK tools.
+
+Besides Docker, all tools downloaded in the xref:policies-pdk-prerequisites.adoc[PDK prerequisites] accept the `HTTP_PROXY` and `HTTPS_PROXY` environment variables.
```
Comment (dhirsch1) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc]: I suggest putting this in first paragraph.

File: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc (hunk 2 above)
Comment (dhirsch1) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc:9]: Consider putting this in the first paragraph it doesn't look too long.

File: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+endif::[]
+:imagesdir: ../assets/images
+
+If you use a network proxy to connect to the internet, it may be necessary to define the network proxy for the PDK tools.
+
+Besides Docker, all tools downloaded in the xref:policies-pdk-prerequisites.adoc[PDK prerequisites] accept the `HTTP_PROXY` and `HTTPS_PROXY` environment variables. 
+
+To configure your environment variable, refer to <<configure-a-network-proxy>>.
+
+To use Docker with a network proxy, also <<configure-a-network-proxy-for-docker>>.
```
Comment (dhirsch1) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc:11]:
```suggestion
To use Docker with a network proxy, also <<configure-a-network-proxy-for-docker,configure a network proxy for Docker>>.
```

File: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+xref:policies-pdk-intergration-testing.adoc[Integration testing] and xref:policies-pdk-debug-local.adoc[the debugging environment] require Docker.
+
+Docker requires access to the internet to pull Docker images. To configure a network proxy for Docker, refer to the following Docker documentation for your relevant use case:
+
+* https://docs.docker.com/network/proxy/[Docker CLI Client]
+* https://docs.docker.com/desktop/settings/mac/#proxies[Docker Desktop on Mac]
+* https://docs.docker.com/desktop/settings/windows/#proxies[Docker Desktop on Windows]
+* https://docs.docker.com/desktop/settings/linux/#proxies[Docker Desktop on Linux]
+
+Docker may visit the following domains during execution:
```
Comment (dhirsch1) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc]: Again, "visit..." doesn't seem like the right word.

**Review by dhirsch1 — APPROVED** [review]

File: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc [hunk 8]
```diff
@@ -0,0 +1,52 @@
+= Configuring a Network Proxy for PDK
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+If you use a network proxy to connect to the internet, you may have to define the network proxy for the PDK tools. Besides Docker, all tools downloaded in the xref:policies-pdk-prerequisites.adoc[PDK prerequisites] accept the `HTTP_PROXY` and `HTTPS_PROXY` environment variables.
```
Comment (dhirsch1) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc]:
```suggestion
If you use a network proxy to connect to the internet, you might have to define the network proxy for the PDK tools. Besides Docker, all tools downloaded in the xref:policies-pdk-prerequisites.adoc[PDK prerequisites] accept the `HTTP_PROXY` and `HTTPS_PROXY` environment variables. 
```

File: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+----
+
+If your network proxy requires authentication, use the following command:
+
+[source,ssh]
+----
+export HTTPS_PROXY=<proxy-method>://<username>:<password>@<proxy-server>:<proxy-port>
+----
+
+The PDK tools may visit the following domains during execution:
```
Comment (dhirsch1) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc]: may > might

Did you think of a better word than "visit"? It is good to respond to a comment if you don't agree with it or if you can't think of a way to resolve it.

**Review by dhirsch1 — APPROVED** [review]

File: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+endif::[]
+:imagesdir: ../assets/images
+
+If you use a network proxy to connect to the internet, you might have to define the network proxy for the PDK tools. Besides Docker, all tools downloaded in the xref:policies-pdk-prerequisites.adoc[PDK prerequisites] accept the `HTTP_PROXY` and `HTTPS_PROXY` environment variables. 
+
+To configure your environment variable, refer to <<configure-a-network-proxy>>.
+
+To use Docker with a network proxy, also <<configure-a-network-proxy-for-docker>>.
+
+NOTE: Depending on your network proxy, you may have to define the domains the PDK tools access in your proxy's allowlist.
```
Comment (dhirsch1) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc]: Glenn, this should be "might" instead of "may."

**Review by dhirsch1 — APPROVED** [review]

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**